service/src/models/archaeologist.ts

```diff
@@ -147,6 +148,7 @@ export class Archaeologist {
                   arweaveTxId,
                   unencryptedShardDoubleHash,
                   maxRewrapInterval,
+                  diggingFee,
```

Comment:
@katien -- the service will need to validate the digging fee is sufficient -- similar to check on line `154` -- the value in memory to validate against is:

`inMemoryStore.profile!.minimumDiggingFee`

---

Also, complete tangent --  we should consider adding a timestamp to the signature that the contracts could validate is within a certain interval, to avoid the embalmer stashing signatures for later use. 

I'm not sure if also adding the `sarcoId` to the sig would add any extra security benefits. 

These items could get handled in a different PR (as would need contract updates as well), but wanted to bring them to discussion.

Reply:
Thanks for catching that missing validation!

PR based on our discussion around timestamps - we'll omit `sarcoId` for now because there's no clear benefit to adding it but if we come up with a reason in the future it's already available in the contracts and easy to change.
https://github.com/sarcophagus-org/sarcophagus-v2-archaeologist-service/issues/41